src/components/Sidebar/Sidebar.jsx

```diff
@@ -19,14 +20,17 @@ export const Sidebar = () => {
     { id: v4(), name: "History", Icon: BsClockHistory },
   ];
 
+  const { open, setOpen } = useData();
+
   return (
-    <aside className="sidebar">
+    <aside className={`sidebar ${open ? "active" : ""}`}>
       <ul className="list full-width">
         {sidebarList.map(({ name, id, Icon }) => (
           <Link
             key={id}
             className={`list-item h3 mx-md my-xs p-xs transition ${styles.sidebar_list_item}`}
-            to="/"
+            to={`/${name === "Home" ? "" : name.toLowerCase()}`}
```

Comment:
Here, if the name of my current list item is `Home` then Link will redirect to `/` where if it's anything else let's say `Explore` it will redirect to `/explore`